src/i18n/locales/en/translation.json

```diff
@@ -66,6 +66,7 @@
           },
           "optional_lightning_address_label": "Choose your Lightning Address (optional)",
           "optional_lightning_address_suffix": "@getalby.com",
+          "optional_lightning_address_title": "numbers and letters, at least 3 characters",
```

Comment:
Should this be uppercase?

Reply:
what? you mean "Numbers..."
I don't know/care.
<img width="583" alt="image" src="https://user-images.githubusercontent.com/318/189632319-848afb76-b823-4db0-9886-d9b209e97a95.png">